news/2019-09-27-mappers-guild-september-newspost.md

```diff
@@ -0,0 +1,146 @@
+---
+layout: post
+title: Mappers' Guild Quests Completed
+date: 2019-09-24 00:00:00 +0000
+---
+
+The Mappers' Guild has been hard at work creating beatmaps for your favourite Featured Artist tracks. If you haven't been following, come check them out!
+
+![](/wiki/shared/news/banners/mappersguild.png)
+
+Since March 2019, the **[Mappers' Guild](https://mappersguild.com)** has been open to everyone having at least three ranked maps. This has made the Guild very productive in terms of mapping and modding. With the introduction of the quest system, mappers have been more active and interested in mapping Featured Artist content than ever before.
```

Comment:
```suggestion
Since March 2019, the **[Mappers' Guild](https://mappersguild.com)** has been open to everyone with at least three ranked maps. This has made the Guild very productive in terms of mapping and modding. With the introduction of the quest system, mappers have been more active and interested in mapping Featured Artist content than ever before.
```